lib/exports/colors-new/colors.less

```diff
@@ -0,0 +1,54 @@
+@import (reference) "./color-mixins.less";
+
+// EXISTING ISSUES
+// - theme primary vars not generated properly 🤷‍♂️ (--_ private var issue via Less)
```

Comment:
@dancormier how confident are we about making those variables "pseudo" private? Do we have enough evidence they won't need to be referred in Core?
What are the consequences of making these variables private from a documentation perspective? [Today we are documenting them as if people could access them](https://stackoverflow.design/product/base/colors/). 

In the spirit of working on iterative small steps, what do you think about dropping the transition to "pseudo" private variable at least for this PR and revaluate at some point down the line (still before releasing 2.0)? 

I am suggesting this also because my understanding is that some of those issues reported here are likely due to that.

Reply:
That comment is outdated and needs to be removed altogether. I've already fixed the issue mentioned there and dropped the pseudo private prefix from the colors and functions but still have a note or two lying around. I'll do a clean up soon.